### DISCUSSION TITLE
Problem with routing to the local network

### DISCUSSION THREAD

**Discussion by enzzo-ux** [strongswan/strongswan#1762] [General]
Something went wrong after restarting the server and I can't find the cause. I suspect there is no manually added firewall rule or manually added route.

the vpn client can ping the server
You can ping the client from the server
from vpn client can't ping other hosts on lan 10.0.10.x and vice versa
there is no nat and masquerade on the server, vpn is only for one ip to have access to the local network.

200.200.200.35 - external ip of the server
200.200.200.55 - client's nat ip
10.0.10.70 - server lan ip
10.0.10.92 - ip client vpn

 SUN_VPN[23]: ESTABLISHED 15 seconds ago, 200.200.200.35[200.200.200.35]...200.200.200.55[C=XX, O=FIRM, CN=200.200.200.35]
     SUN_VPN[23]: IKEv2 SPIs: 578619d00a184571_i 6ae1bf17e89a2bc0_r*, public key reauthentication in 2 hours
     SUN_VPN[23]: IKE proposal: AES_CBC_128/HMAC_SHA2_256_128/PRF_HMAC_SHA2_256/ECP_256
     SUN_VPN{14}:  INSTALLED, TUNNEL, reqid 1, ESP in UDP SPIs: c815af06_i cb4eb491_o
     SUN_VPN{14}:  AES_CBC_128/HMAC_SHA2_256_128, 0 bytes_i, 0 bytes_o, rekeying in 43 minutes
     SUN_VPN{14}:   10.0.10.0/24 === 10.0.10.92/32

ip route show table 220
10.0.10.92 via 200.200.200.55 dev br0 proto static src 10.0.10.70 mtu 1360
net/ipv4/ip_forward=1
iptables -t nat -I POSTROUTING -m policy --pol ipsec --dir out -j ACCEPT

what could be missing?

**Comment by tobiasbrunner** [comment]
> SUN_VPN{14}: 10.0.10.0/24 === 10.0.10.92/32

If you assign virtual IPs from your local subnet, you have to fake ARP responses for your clients e.g. via _farp_ plugin (see [the docs](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html)).

**Reply by enzzo-ux** [reply]
Thanks
